sktime/forecasting/chronos.py

```diff
@@ -293,7 +293,7 @@ class ChronosForecaster(_BaseGlobalForecaster):
         ],
         # abdulfatir and lostella for amazon-science/chronos-forecasting
         "maintainers": ["geetu040"],
-        "python_dependencies": ["torch", "transformers", "accelerate"],
+        "python_dependencies": ["torch", "transformers", "accelerate", "numpy==1.26.4"],
```

Comment:
I assume this is just diagnostic? Because we cannot merge it like that, that would restrict `numpy` to a pin on all operating systems.